src/App.js

```diff
@@ -1,25 +1,49 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+
+import "./App.css";
+
+const App = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [changeBackgroundColor, setChangeBackgroundColor] = useState(true);
```

Comment:
changeBackgroundColor sounds like a function - like it changes the backgroundColor, but it's just a boolean value that sets color to white if true.. 
I'd rename it to something like "isMainBackgroundColor"